docs/dev/checks/builtins/_util.md

```diff
@@ -0,0 +1,39 @@
+# _util
+
+> **This module is intended for internal use and should never be imported directly.**
+> Checks should use the methods exposed by the `AgentCheck` class instead, see
+> [dedicated docs](https://datadog-checks-base.readthedocs.io/en/latest/) for
+> more details.
+
+The module exposes low level functions to run processes from Python integrations.
+
+## Implementation
+
+* [_util.c](/six/common/builtins/_util.c)
+* [_util.h](/six/common/builtins/_util.h)
+* [util.go](/pkg/collector/python/util.go)
+
+## Functions
+
+```python
+def subprocess_output(args, raise_on_empty):
+    """Run an external process and return the output.
+
+    NOTE: If unicode is passed to any of the params accepting it, the
```

Comment:
This note is for Python2 only: do we want to update the doc for Python3 later ?

Reply:
We can change these docs once Python3 becomes the default, in this PR I wanted to keep things easy so that newcomers don't get confused.